Mathlib/Combinatorics/SimpleGraph/Hasse.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.Data.Fin.SuccPred
 import Mathlib.Data.Nat.SuccPred
 import Mathlib.Order.SuccPred.Relation
 import Mathlib.Tactic.FinCases
+import Mathlib.Data.ZMod.Defs
```

Comment:
Accident?

Reply:
This was due to a `lake exe shake --fix` after this [failed build](https://github.com/leanprover-community/mathlib4/actions/runs/10009285873/job/27667845563). Looks like it isn't necessary indeed, so I removed it.

Reply:
Oh yeah I see